config/locales/en.yml

```diff
@@ -72,7 +72,6 @@ en:
   manage_achievements: "Manage achievements"
   achievement_title: "Title"
   achievement_description: "Description"
-  achievement_points: "Points"
```

Comment:
My bad for not commenting a change that isn't immediately obvious: the `achievement_points` label was duplicated, see `#122`; I merely removed the dupe.